site/src/pages/DeploymentSettingsPage/NotificationsPage/NotificationsPage.tsx

```diff
@@ -30,10 +29,12 @@ export const NotificationsPage: FC = () => {
 			notificationDispatchMethods(),
 		],
 	});
-	const ready =
-		templatesByGroup.data && dispatchMethods.data && deploymentValues;
-	const tab = searchParams.get("tab") || "events";
+	const tabState = useSearchParamsKey({
+		key: "tab",
+		defaultValue: "events",
+	});
 
+	const ready = !!(templatesByGroup.data && dispatchMethods.data);
```

Comment:
nit: prefer `Boolean(…)` over `!!(…)`, it makes it much clearer that you want a boolean cast

Reply:
So do I, but for some reason, when I tried to change things, it wasn't doing anything with type narrowing